forms-web-app/src/controllers/eligibility.js

```diff
@@ -19,7 +19,7 @@ exports.postDecisionDate = (req, res) => {
   const month = req.body['decision-date-month'];
   const year = req.body['decision-date-year'];
 
-  const date = moment(`${year}-${month}-${day}`, 'Y-M-D', true);
+  const date = moment(`${year}-${month}-${day}`, 'YYYY-M-D', true);
```

Comment:
I know moment was done before your time (so this is just a comment, not a request to change), but just noticed the [project status](https://momentjs.com/docs/)

> We now generally consider Moment to be a legacy project in maintenance mode. It is not dead, but it is indeed done.

It may well be worth a future piece of work to replace moment with either the native `Date` or `Intl` class or [Luxon](https://moment.github.io/luxon/) (the interface is similar as it's made by the same people)

Reply:
Ok I keep this in mind, thanks for the info.